src/gui/mainwindow.cpp

```diff
@@ -878,7 +878,7 @@ void MainWindow::createKeyboardShortcuts()
 // Keyboard shortcuts slots
 void MainWindow::displayTransferTab() const
 {
-    m_tabs->setCurrentWidget(m_transferListWidget);
```

Comment:
I believe it doesn't work because `m_transferListWidget` doesn't represent the tab directly but it's `m_splitter` that does.

Reply:
That works too.